active/website.md

```diff
@@ -0,0 +1,62 @@
+# Website Working Group
+
+## Scope of responsibilities
+
+This working group will manage Django's official website, **djangoproject.com** on behalf of the Django Software Foundation
+
+The duties of the working group are:
+- Introduce new features on the website
+- Maintain and monitor the website 
+- Ensure that information on the website is accurate 
+
+
+### Delegated responsibilities:
+- Chair, Co-chair and Board Liaison can make final decision of  sections of the website that might not conform with DSF's CoC.
+- Members can work on bugs
+- Members can tag,label and respond to issues.
+- Member can maintain and monitor the website including updating versions
+- Mentor new contributors to the website
+- Chair, Co-Chair and Board Liaison can sign off on new features
+
+## Initial membership
+
+- Chair: TBA
+- Co-Chair: TBA
+- Board Liaison (must be an active Board member; may be the same as Chair/Co-Chair): Thibaud Colas
+- Other members:
+    - Jacob Kaplan-Moss
+    - Sarah Abderemane
+    - Eric Sherman
+    - Mark Walker
+    - Jason Judkins
+    - Paolo Melchiorre
+    - Sanyam Khurana
+    - Tobias McNulty
+    - Ron Maravanyika
+
+
+
+## Future membership
+
+### Who is eligible to join? Any volunteer, or are there specific requirements?
+
+Members must have interest in Django and should be able to work with Django. Most importantly members must be willing to adhere to Django's [Code of Conduct](https://www.djangoproject.com/conduct/) . Members must be well versed with the process of contributing to **djangoproject.com** or at least willing to be guide. We welcome all experience levels, we also welcome first time contributors. 
+
+### How do people who want to join sign up / volunteer / express interest?
+TBA
+
+### How will decisions on adding/removing members be handled?
+Direct membership: new members may self-nominate; the WG will vote (50%+1) to approve/deny new members; the WG will directly vote on new Chair/Co-Chairs.
+
+Members join the group for a 2 year term. At the end of this term, they need to opt into staying involved to keep being 
+a member of the group.
+
+
+## Budget
+TBA
+
+## Comms
+- A mailing list that we'll create, `website-wg@djangoproject.com`
```

Comment:
Could you explain why you’re proposing a mailing list? They tend to run slow in my experience and are prone to spam. Django has a Discord server where we could have a private channel. The DSF has a Slack workspace. Could we use either of those?

Reply:
Well this is open to discussion, maintainers can decided where they want comms to be. I just remembered that l got involved on the site through the DSF individual members mailing list so i just assumed.

Reply:
Is there a DSF slack?

Reply:
I see we now have both the Slack channel and a mailing list. @ronnzw I think you need to define how each of those would be used, or if it’s open to discussion then explicitly write it here.

Keep in mind the more clear-cut the group charter is, the easier it is for people to consider joining and for the board to review. So if discussions need to happen, now is the best time to raise this question with interested members.

---

@jcjudkins yes, the DSF has a Slack Workspace for internal matters. Largely this has been used for paid employees, the board of directors, and a few working groups now.

Reply:
IMHO slack works fine for the working group. Mailing list can be for discussion among larger audience.

Reply:
@ronnzw I think this one is the only missing update from your part according Thibaud's message

Reply:
I see "DSF slack" is now there so I’d just recommend removing the point about the mailing list and that’s it :)

Reply:
Is everyone happy with me removing the mailing list? Despite a mailing list being included in the WG guidelines?

Reply:
I can see a mailing list being useful as a way of making announcements and interacting with the wider audience. But if we feel that might not be as necessary for Website WG, then removing the point about mailing list makes sense.

Reply:
We discussed this at today’s DjangoCon US sprint meeting, with @jacklinke @tobiasmcnulty @pauloxnet @CuriousLearner @jcjudkins @bmispelon @thibaudcolas @jbeimler. Almost everyone in this group already uses Slack and will be happy there. No objections to having a mailing list in addition if this feels helpful.